stingray/lightcurve.py

```diff
@@ -156,8 +156,9 @@ def __init__(self, time, counts, err=None, input_counts=True,
                 err_low, err_high = poisson_conf_interval(np.asarray(counts),
                     interval='frequentist-confidence', sigma=1)
                 # calculate approximately symmetric uncertainties
-                err = (np.absolute(err_low) + np.absolute(err_high) -
-                       2 * np.asarray(counts))/2.0
+                err_low -= np.asarray(counts)
```

Comment:
@dhuppenkothen the old function was returning values like 0.5 when the actual value was 100. Do you have any idea why 
```
(np.absolute(err_low) + np.absolute(err_high) - 2 * np.asarray(counts))/2.0
```
should be different from
```
err_low -= np.asarray(counts)
err_high -= np.asarray(counts)
err = (np.absolute(err_low) + np.absolute(err_high))/2.0
```
? It was driving me mad